desktop/apps/partner2/stylesheets/overview.styl

```diff
@@ -7,6 +7,7 @@
     margin-bottom 60px
     border-top 1px solid gray-color
     flex 0 1 100%
+    width 100%
```

Comment:
Nice! 

My only question is whether it makes sense to specify `width` and `flex-basis`, i.e. the third argument to the `flex` rule. I _think_ `width` strictly overrides it?

In any case this looks legit according to my manual cross-browser inspection. Thanks!